inception-project/inception
# #3194 - Jumping to location may not bring target into view

**What's in the PR**
- Track the edit highlight or the match focus and try to keep them in view after rendering

**How to test manually**
* Use the brat editor in different cases for annotating and curating and see if the scrolling interferes with normal work or if it helps

**Automatic testing**

**Documentation**
